src/clarity-angular/datagrid/datagrid-action-overflow.ts

```diff
@@ -4,49 +4,41 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 import {
-    Component, EventEmitter, HostListener, Input, Output, ElementRef, ViewChildren, QueryList, AfterViewInit, OnDestroy
+    Component, EventEmitter, HostListener, Input, Output, ElementRef, ViewChild
 } from "@angular/core";
-import {Subscription} from "rxjs/Subscription";
-import {RowActionService} from "./providers/row-action-service";
-import {Popover, Direction} from "../popover/popover";
+import {Point, PopoverOptions} from "../popover/popover";
+import {DatagridRenderOrganizer} from "./render/render-organizer";
 
 
 @Component({
     selector: "clr-dg-action-overflow",
     template: `
-        <clr-icon shape="ellipsis-vertical" class="datagrid-action-toggle" (click)="toggle()"></clr-icon>
-        <div #menu class="datagrid-action-overflow" *ngIf="open">
+        <clr-icon #anchor shape="ellipsis-vertical" class="datagrid-action-toggle" (click)="toggle()"></clr-icon>
+        <div #menu class="datagrid-action-overflow" *clrPopover="open; anchor: anchor; anchorPoint: anchorPoint; 
+            popoverPoint: popoverPoint; options: popoverOptions;">
             <ng-content></ng-content>
         </div>
     `
 })
 
-export class DatagridActionOverflow implements OnDestroy, AfterViewInit {
+export class DatagridActionOverflow {
 
-    constructor(private elementRef: ElementRef, private rowActionService: RowActionService) {}
+    public anchorPoint: Point = Point.RIGHT_CENTER;
+    public popoverPoint: Point = Point.LEFT_CENTER;
+    public popoverOptions: PopoverOptions = { useAnchorParent: true, offsetX: 5 };
```

Comment:
I got away without using popoverOptions all together here by using the margin as per your suggestion.